spec/requests/api/framework_responses_controller_bulk_update_spec.rb

```diff
@@ -7,11 +7,14 @@
     include_context 'with supplier with spoofed access token'
 
     subject(:bulk_update_framework_responses) do
+      Timecop.freeze(recorded_timestamp)
```

Comment:
Perhaps use block version instead? It feels more robust somehow than adding the explicit `Timecop.return` line 🤔 

```suggestion
      Timecop.freeze(recorded_timestamp) do
        ...
      end
```